apps/sertra/pages/server/[id].tsx

```diff
@@ -44,6 +46,13 @@ export const getStaticProps: GetStaticProps<
 const ServerDetail: React.FC<{ detail_listing: ServerListingDetailData }> = ({
   detail_listing,
 }) => {
+  const [showPopup, setShowPopup] = useState(false);
+  const togglePopup = () => {
+    setShowPopup(!showPopup);
```

Comment:
I'm not sure how important it's in this particular case, but usually I'd do this as `setShowPopup((current) => !current);`, which guarantees that the toggling is done to the latest value of `showPopup` in the state and not some outdated value.